builder/virtualbox/ovf/config.go

```diff
@@ -101,6 +102,13 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 		if err != nil {
 			errs = packer.MultiErrorAppend(errs, fmt.Errorf("source_path is invalid: %s", err))
 		}
+		// file must exist now.
+		if (len(c.SourcePath) > 7) && (c.SourcePath[:7] == "file://") {
```

Comment:
my preference for this is 

```
    fileURL, _ := url.Parse("file:///var/foo.baz")
    if fileURL.Scheme == "file" {
        fmt.Println(fileURL.Path)
    }
```

Reply:
I'm fine with changing your structure here.  I think I'd rather keep DownloadableURL's returns how they are for now, though.